client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -0,0 +1,230 @@
+			'stats-module',
+			'is-expanded',
+			'summary',
+			'is-followers-page',
+			{
+				'is-loading': this.props.followersList.isLoading(),
+				'is-showing-info': this.state.showInfo,
+				'has-no-data': noData,
+				'is-showing-error': hasError || noData
+			}
+		];
+
+		switch ( this.props.followType ) {
+			case 'comment':
+				itemType = this.translate( 'Comments' );
+				break;
+
+			case 'email':
+				itemType = this.translate( 'Email' );
+				break;
+
+			case 'wpcom':
+				itemType = this.translate( 'WordPress.com' );
+				break;
+		}
+
+		if ( data.total ) {
+			startIndex = this.props.perPage * ( this.props.page - 1 ) + 1;
+			endIndex = this.props.perPage * this.props.page;
+
+			if ( endIndex > data.total ) {
+				endIndex = data.total;
+			}
+
+			paginationSummary = this.translate( 'Showing %(startIndex)s - %(endIndex)s of %(total)s %(itemType)s followers', {
+				context: 'pagination',
+				comment: '"Showing [start index] - [end index] of [total] [item]" Example: Showing 21 - 40 of 300 WordPress.com followers',
+				args: {
+					startIndex: this.numberFormat( startIndex ),
+					endIndex: this.numberFormat( endIndex ),
+					total: this.numberFormat( data.total ),
+					itemType: itemType
+				}
+			} );
+
+			paginationSummary = (
+				<div className="module-content-text module-content-text-stat">
+					<p>{ paginationSummary }</p>
+				</div>
+			);
+		}
+
+		pagination = <Pagination page={ this.props.page } perPage={ this.props.perPage } total={ data.total } pageClick={ this.props.pageClick } />;
+
+		if ( data && data.posts ) {
+			followers = <StatsList data={ data.posts } moduleName="Followers" />;
+			labelLegend = this.translate( 'Post', {
+				context: 'noun'
+			} );
+			valueLegend = this.translate( 'Followers' );
+		} else if ( data && data.subscribers ) {
+			followers = <StatsList data={ data.subscribers } followList={ this.props.followList } moduleName="Followers" />;
+			labelLegend = this.translate( 'Follower' );
+			valueLegend = this.translate( 'Since' );
+		}
+
+		moduleHeaderTitle = (
+			<h4 className="module-header-title">{ this.translate( 'Followers' ) }</h4>
+			);
+
+		if ( 'email' === this.props.followType ) {
+			emailExportUrl = 'https://dashboard.wordpress.com/wp-admin/index.php?page=stats&blog=' + this.props.site.ID + '&blog_subscribers=csv&type=email';
+
+			emailExportLink = (
+				<div className="module-content-text">
+					<ul className="documentation">
+						<li>
+							<a href={ emailExportUrl } target="_blank" onClick={ this.recordDownloadClick }>
+								<Gridicon icon="cloud-download" />
+								{ this.translate( 'Download all email followers as CSV', { context: 'Action shown in stats followers module to download all email followers.' } ) }
+							</a>
+						</li>
+					</ul>
+				</div>
+			);
+		}
+
+		return (
+			<Card className={ classNames.apply( null, classes ) }>
+				<div className="followers">
+					<div className="module-header">
+						{ moduleHeaderTitle }
```

Comment:
Is there a reason not to simply include the content of this element here verses pulling it in as a separate variable? I think it would be easier to see what's being rendered if the `<h4 />` were inserted at this point.